bot-lib/src/courses.rs

```diff
@@ -19,10 +19,10 @@ static COURSES: LazyLock<DashMap<String, Course>> = LazyLock::new(|| {
 
     tracing::info!("Loaded {} courses", courses.len());
 
-    courses
+    courses 
```

Comment:
Fix this whitespace change

Reply:
This is the `cargo fmt` I mentioned. No need for an extra comment, resolving.